galadriel/dashboard/state.py

```
@@ -1,146 +1,193 @@
+
+        updated_cases = session.exec(
+            select(IterationSnapshotModel).where(
+                IterationSnapshotModel.child_type == consts.CHILD_TYPE_STEP,
+                IterationSnapshotModel.updated >= cutoff_date,
+            ).order_by(desc(IterationSnapshotModel.updated))
+        ).all()
+
+        for updated_case in updated_cases:
+            case_date = updated_case.updated.strftime("%Y-%m-%d")
+            for trend_entry in trend_data:
+                trend_entry_date = datetime.strptime(trend_entry["date"], "%Y-%m-%d %H:%M:%S")
```

Ruff (0.15.4)

[warning] Naive datetime constructed using `datetime.datetime.strptime()` without %z

(DTZ007)